en/02_Developer_Guides/14_Files/02_Images.md

```diff
@@ -334,6 +334,20 @@ SilverStripe\Core\Injector\Injector:
 
 Manipulated images are stored as "file variants" in the same folder structure as the original image. The storage mechanism is described in the ["File Storage" guide](file_storage).
 
+## Controlling how images are rendered
+
+Developers can customise how `Image` instances are rendered on their website by overriding the the `templates/SilverStripe/Assets/Storage/DBFile_Image.ss` template file.
+
+This will apply to images added to an HTMLEditorField and SS template images.
+
+You can also choose to have different rendering logic for HTMLEditorField images and for SS templates images by overriding different templates.
+
+- Add a `SilverStripe/Assets/Shortcodes/ImageShortcodeProvider_Image.ss` to your theme to control images added to an HTMLEditorField.
+- Add a `DBFile_Image.ss` file to the root of your theme to control only SS templates images.
```

Comment:
```suggestion
- Add a `DBFile_Image.ss` file to the root of your theme to control only SS template images.
```

Reply:
I used Guy's "invoked in templates" language instead.